geode-wan/src/test/java/org/apache/geode/management/internal/cli/commands/DescribeRegionDUnitTest.java

```diff
@@ -0,0 +1,165 @@
+
+import org.apache.geode.cache.Cache;
+import org.apache.geode.cache.CacheFactory;
+import org.apache.geode.cache.EvictionAction;
+import org.apache.geode.cache.EvictionAttributes;
+import org.apache.geode.cache.FixedPartitionAttributes;
+import org.apache.geode.cache.PartitionAttributes;
+import org.apache.geode.cache.PartitionAttributesFactory;
+import org.apache.geode.cache.Region;
+import org.apache.geode.cache.RegionFactory;
+import org.apache.geode.cache.RegionShortcut;
+import org.apache.geode.test.dunit.rules.LocatorServerStartupRule;
+import org.apache.geode.test.dunit.rules.MemberVM;
+import org.apache.geode.test.junit.categories.DistributedTest;
+import org.apache.geode.test.junit.rules.GfshCommandRule;
+
+@Category(DistributedTest.class)
+public class DescribeRegionDUnitTest {
+  private static final String REGION1 = "region1";
+  private static final String REGION2 = "region2";
+  private static final String REGION3 = "region3";
+  private static final String SUBREGION1A = "subregion1A";
+  private static final String SUBREGION1B = "subregion1B";
+  private static final String SUBREGION1C = "subregion1C";
+  private static final String PR1 = "PR1";
+  private static final String LOCAL_REGION = "LocalRegion";
+
+  private static final String PART1_NAME = "Par1";
+  private static final String PART2_NAME = "Par2";
+
+  @ClassRule
+  public static LocatorServerStartupRule lsRule = new LocatorServerStartupRule();
+
+  @ClassRule
+  public static GfshCommandRule gfsh = new GfshCommandRule();
+
+  @BeforeClass
+  public static void setupSystem() throws Exception {
+    Properties props = new Properties();
+    props.setProperty(DISTRIBUTED_SYSTEM_ID, "" + 1);
+    MemberVM sending_locator = lsRule.startLocatorVM(1, props);
+
+    props.setProperty(DISTRIBUTED_SYSTEM_ID, "" + 2);
+    props.setProperty(REMOTE_LOCATORS, "localhost[" + sending_locator.getPort() + "]");
+    lsRule.startLocatorVM(2, props);
+
+    MemberVM server1 = lsRule.startServerVM(3, "group1", sending_locator.getPort());
+    MemberVM server2 = lsRule.startServerVM(4, "group2", sending_locator.getPort());
+
+    configureServers(server1, server2);
+
+    gfsh.connectAndVerify(sending_locator);
+    gfsh.executeAndAssertThat("create async-event-queue --id=queue1 --group=group1 "
+        + "--listener=org.apache.geode.internal.cache.wan.MyAsyncEventListener").statusIsSuccess();
+    gfsh.executeAndAssertThat("create gateway-sender --id=sender1 --remote-distributed-system-id=2")
+        .statusIsSuccess();
+    sending_locator.waitTillAsyncEventQueuesAreReadyOnServers("queue1", 1);
+    sending_locator.waitTilGatewaySendersAreReady(2);
+
+    gfsh.executeAndAssertThat(
+        "create region --name=region4 --type=REPLICATE --async-event-queue-id=queue1 --gateway-sender-id=sender1")
+        .statusIsSuccess();
+
+  }
+
+  @SuppressWarnings("deprecation")
+  private static void configureServers(MemberVM server1, MemberVM server2) {
```

Comment:
just for easy understanding of what this test is doing, we could probably get rid of this entire call. We are not describing any of the regions created in here, right?

Reply:
Sure could!  Another instance of not being judicious in what I copy when pulling something from another class.